src/WebWindow.Native/WebWindow.Native.vcxproj

```diff
@@ -159,14 +159,14 @@
   </ItemGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
-    <Import Project="..\..\packages\Microsoft.Web.WebView2.0.8.270\build\native\Microsoft.Web.WebView2.targets" Condition="Exists('..\..\packages\Microsoft.Web.WebView2.0.8.270\build\native\Microsoft.Web.WebView2.targets')" />
-    <Import Project="..\..\packages\Microsoft.Windows.ImplementationLibrary.1.0.190716.2\build\native\Microsoft.Windows.ImplementationLibrary.targets" Condition="Exists('..\..\packages\Microsoft.Windows.ImplementationLibrary.1.0.190716.2\build\native\Microsoft.Windows.ImplementationLibrary.targets')" />
+    <Import Project="..\..\packages\Microsoft.Web.WebView2.0.8.355\build\native\Microsoft.Web.WebView2.targets" Condition="Exists('..\..\packages\Microsoft.Web.WebView2.0.8.355\build\native\Microsoft.Web.WebView2.targets')" />
```

Comment:
After this change, it no longer is able to run on my Windows installation. The call to `CreateWebView2EnvironmentWithDetails` returns a failure with message `The system cannot find the file specified.`

My guess is that the newer version of the package only works if you have a non-beta version of Edge-Chromium installed, or if you explicitly pass a path to the beta binaries. Maybe there is some other startup API that works with both beta and non-beta installations; I'm not sure.

Reply:
I updated the packages because the old ones don't work with the new version of Edge Dev. I think WebView2 is following the Dev version, but not Beta.

Reply:
OK, thanks for checking. I suspect we need it to work with Beta by default, not Dev, since more people using this will have that version.

We could maintain a branch in which it uses the Dev version for testing, but the `master` branch and published packages should retain compatibility with Beta.

Does that seem reasonable?

Reply:
That's fair, thanks!